### PR TITLE
Create datadog-synthetics.yml

### PR DESCRIPTION
**Pull request by OKEAMAH**

### CODE REVIEW

**Review by OKEAMAH — COMMENTED** [review]
App

**Review by OKEAMAH — COMMENTED** [review]
B

File: .github/workflows/datadog-synthetics.yml
Comment (OKEAMAH) [inline .github/workflows/datadog-synthetics.yml:1]: This

**Review by Sturgeontara06 — COMMENTED** [review]
K

File: .github/workflows/datadog-synthetics.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    # Run Synthetic tests within your GitHub workflow.
+    # For additional configuration options visit the action within the marketplace: https://github.com/marketplace/actions/datadog-synthetics-ci
+    - name: Run Datadog Synthetic tests
+      uses: DataDog/synthetics-ci-github-action@2b56dc0cca9daa14ab69c0d1d6844296de8f941e
```
Comment (Kudlaty955) [inline .github/workflows/datadog-synthetics.yml:32]: Git

**Review by Kudlaty955 — APPROVED** [review]

File: .github/workflows/datadog-synthetics.yml (hunk 1 above)
Comment (Kudlaty955) [reply .github/workflows/datadog-synthetics.yml:32]: Uni 3

**Review by Kudlaty955 — APPROVED** [review]

**Pull request closed without merging**